networks.yaml

```
@@ -71,6 +71,9 @@ kovan:
   Gyro2PoolFactory:
     address: "0xba9519ce545aEe8d4Df51a3914CC22AB85E90CeA"
     startBlock: 30120881
+  Gyro3PoolFactory:
+    address: "0xDB42004e392aCdeAD8A1138E8AdE52E4c801a8F8"
+    startBlock: 30827386
```

Could you update this, run `yarn codegen`, and commit the updated `subgraph.kovan.yaml`? Thanks 🙏  

```suggestion
    address: "0xBBfD645Be37037bE6c60D498E798037bd8d805d3"
    startBlock: 31238127
```